### DISCUSSION TITLE
Coverage files in tcl?

### DISCUSSION THREAD

**Discussion by Blebowski** [nickg/nvc#684] [General]
Hello,

do you think it would be good to re-work the parsing of Coverage exclude files and possibly Coverage specification files
to be TCL  syntax files (since TCL shell is again supported) ? An advantage would be e.g. possibility to specify common
prefix of excluded path by a variable, e.g.

```
exclude ${IP_ROOT_HIER}/controller_fsm/_S10
```

**Comment by nickg** [comment]
Yes, probably. I want to do the same thing with the include/exclude patterns for waveform dumps. But note that the TCL support is optional at the moment and disabled by default unless you pass `--enable-tcl` to `configure`. That will change in the future once it's more stable/useful.

**Comment by Blebowski** [comment]
I see, so I guess the change can wait until the TCL shell will be required dependency of NVC.
